src/bootstrap/bootstrap.py

```diff
@@ -669,7 +670,7 @@ def bootstrap():
     rb.update_submodules()
 
     # Fetch/build the bootstrap
-    rb.build = rb.build_triple()
+    rb.build = args.build or rb.build_triple()
```

Comment:
It doesn't. The command line is handled in the `bootstrap()` function. The parsed `args`, or stuff derived from them, are set as attributes of the `RustBuild` object.